modules/op-release-notes-1-6.adoc

```diff
@@ -306,4 +306,32 @@ Error from server (InternalError): Internal error occurred: failed calling webho
 ** For implicit params:
 *** Do not apply the `PipelineSpec` parameters to the `TaskRefs` object.
 *** Disable implicit param behavior for the `Pipeline` objects.
-// https://github.com/tektoncd/pipeline/releases/tag/v0.28.3
\ No newline at end of file
+// https://github.com/tektoncd/pipeline/releases/tag/v0.28.3
+
+[id="release-notes-1-6-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.6.3
+
+[id="deprecated-features-1-6-3_{context}"]
+=== Deprecated features
+
+* With this update, the {pipelines-title} Operator stops installing pod security policies from components, as the policies are deprecated. As a result, the following upgrade paths are affected:
+** When you upgrade from {pipelines-shortname} 1.6.1 or 1.6.2 to {pipelines-shortname} 1.6.3, the existing pod security policies from components such as Pipelines and Triggers are deleted.
+** When you upgrade from {pipelines-shortname} 1.5.x to 1.6.3, the existing pod security policies are retained. However, cluster administrators can delete them manually.
```

Comment:
Ensure you follow this guideline mentioned here for Deprecated functionality: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes

Reply:
This is not exactly a deprecated feature. This is actually a fixed issue that was overdue for fix after something was deprecated in an earlier release. Accordingly, I have restructured the sentence and included it under Fixed issues.

Reply:
@sounix000 Thanks for the explanation. this helps :)